contracts/src/v0.8/tests/FunctionsOracleMigration.sol

```diff
@@ -0,0 +1,272 @@
+// SPDX-License-Identifier: MIT
```

Comment:
For reviewing purposes the only difference is the fee on line 149 returns 1 instead of 0.

Reply:
SGTM.  Out of curiosity, what is the reason for changing this to 1 instead of 0?